docs/getting-started.md

```diff
@@ -0,0 +1,109 @@
+# Getting started
+
+## Prerequisites
+
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+1. Install `kustomize` `v3.1.0+` (see [kustomize/releases](https://github.com/kubernetes-sigs/kustomize/releases).
```

Comment:
You might have missed a closing bracket here.

Reply:
Yup

Reply:
+1